### ISSUE TITLE
Fortiguard identifies and blocks Wireguard

### ISSUE THREAD

**Issue by TarekSalama** [trailofbits/algo#1586]
So I was lucky enough to be able to test Wireguard with Fortiguard installed on the network.
Fortiguard was able to completely disable Wireguard.

I did notice however that it sometimes identified it as a proxy (Wireguard) and sometimes it identified it as an AWS app (where I host it). I did notice also that it sometimes worked and sometimes it worked only after reconnecting to the network. I couldn't find an exact pattern.

Is there a way to completely hide it from getting identified by Fortiguard? Any idea why it was sometimes identified as a proxy/AWS app?

**Comment by TC1977** [comment]
What if you change the Wireguard port to 1194 (Open VPN) or 443 (https)? Or something completely random?

**Comment by TarekSalama** [comment]
I am not really experienced with networking but is it possible to change it using Algo?
I know I can probably change it in the server script configuration file but would I need to change anything in the device config files that get auto generated?

**Comment by dguido** [comment]
Hi @TarekSalama,

Thank you for reporting this WireGuard issue. Since you reported this, WireGuard has reached stable 1.0.0 release (Spring 2020) and has undergone substantial security review.

WireGuard is now:
- Officially stable and production-ready
- Included in the Linux kernel
- Actively maintained with regular updates
- The recommended VPN protocol for Algo

The issues you experienced were likely related to the pre-1.0 development versions. These have been resolved in the stable releases.

Since WireGuard has matured significantly and this issue relates to early development versions, I'm closing it. If you encounter any issues with current versions of WireGuard and Algo, please feel free to open a new issue.

Thank you for your patience and for using Algo!
